### PR TITLE
Release v1.4.2

### PR DESCRIPTION
**Pull request by varshaprasad96**
**Description of the change:**
Release v1.4.2

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Review by estroz — APPROVED** [review]
/lgtm

File: changelog/generated/v1.4.2.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## v1.4.2
+
+### Bug Fixes
+
+- Bump cryptography dependency in ansible-operator to 3.3.2. This will fix a potential security bug related to update() calls. Ref: https://github.com/pyca/cryptography/blob/main/CHANGELOG.rst#332---2021-02-07. ([#4531](https://github.com/operator-framework/operator-sdk/pull/4531))
```
Comment (estroz) [inline changelog/generated/v1.4.2.md:5]: @varshaprasad96 there should be more entries than just this one right? https://github.com/operator-framework/operator-sdk/pull/4534/files#diff-3d4efde7d2c494084cc25d283c8a1901dd42a258759ccb7f1534ad8d9ac35196

Comment (varshaprasad96) [reply changelog/generated/v1.4.2.md]: yeah, missed this!

**Review by estroz — APPROVED** [review]
/lgtm

**Pull request merged**